docs/workflows/common-workflow-steps/build-and-test/authanticate-with-netrc.md

```diff
@@ -1,18 +1,19 @@
 ---
-title: Authanticate with Netrc
-metaTitle: Authanticate with Netrc
-metaDescription: Authanticate with Netrc
+title: Authenticate with Netrc
```

Comment:
We should also fix typo in the file name.

Do not forget to add redirect rule for the older url since this has been released before and might have been crawled by search engines.